apps/condo/domains/banking/constants.js

```diff
@@ -27,6 +23,14 @@ const BANK_SYNC_TASK_STATUS = {
 const EXPENSES_GROUPED_BY_CATEGORY_AND_COST_ITEM = 'expenses_grouped_by_category_and_cost_item'
 const BANK_ACCOUNT_REPORT_TEMPLATE_VALUES = [EXPENSES_GROUPED_BY_CATEGORY_AND_COST_ITEM]
 
+// Errors
+const INCORRECT_PROPERTY_ID = 'INCORRECT_PROPERTY_ID'
+const EMPTY_BANK_ACCOUNT_REQUEST_EMAIL_TARGET_VALUE = 'EMPTY_BANK_ACCOUNT_REQUEST_EMAIL_TARGET_VALUE'
+const INCORRECT_DATE_INTERVAL = 'dateTo cannot be later than dateFrom'
```

Comment:
You can use translations feature of GQLError to present error message in human readable format